test/cluster/object_store/test_backup.py

```
@@ -1190,6 +1190,62 @@ async def test_restore_tablets_with_different_tablet_hints(build_mode: str, mana
+        tid = await manager.api.restore_tablets(servers[0].ip_addr, ks, 'test', snap_name, servers[0].datacenter, object_storage.address, object_storage.bucket_name, manifests)
+        await manager.api.wait_for_injection_enter(servers[0].ip_addr, "pause_tablet_restore", deadline=time.time() + 120)
+
+        # The node driving the restore dies after the tablet hints were pinned to
+        # the manifest tablet count.
+        await manager.server_stop(servers[0].server_id, convict=True)
+        with pytest.raises(aiohttp.client_exceptions.ClientConnectorError):
+            await manager.api.wait_task(servers[0].ip_addr, tid)
+
+        tid = await manager.api.restore_tablets(servers[1].ip_addr, ks, 'test', snap_name, servers[1].datacenter, object_storage.address, object_storage.bucket_name, manifests)
+        status = await asyncio.wait_for(manager.api.wait_task(servers[1].ip_addr, tid), timeout=60)
+        logger.info(f'Re-issued restore finished with: {status}')
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)